test/plugins/scenarios/go-redisv9/main.go

```diff
@@ -101,6 +107,19 @@ func TestPipelineSetAndGet(ctx context.Context) error {
 }
 
 func main() {
+	connectSingleRedis()
+	connectClusterRedis()
+
+	http.HandleFunc("/execute", executeHandler)
+
+	http.HandleFunc("/health", func(writer http.ResponseWriter, request *http.Request) {
+		writer.WriteHeader(http.StatusOK)
```

Comment:
Is this needed?I believe if you don't do anything it will also return 200.

Reply:
I noticed that there is a "/health" route written in each plugin file. This is probably used to check the health status of the service, right?

Reply:
totally, what I am saying is that you don't need to write 200.

Reply:
I understand. If I don't register this "/health" route, it indeed returns a 200 status code.I will delete this route register.

Reply:
@JamesSunXX Please don't delete the `/health` route, we need to access it to check the application has been set up success. @jcchavezs means you don't need to write `writer.WriteHeader(http.StatusOK)`(only this line).